<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="welcomePage">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>340</y>
        <width>201</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnWelcome_play">
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnWelcome_showRules">
         <property name="text">
          <string>Game rules</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnWelcome_exit">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="ImgDiaballik">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>90</y>
        <width>451</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../../../resources.qrc">:/resources/images/diaballik_logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="gameOptionsPage">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>250</y>
        <width>331</width>
        <height>211</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>23</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="title">
       <string>Game options</string>
      </property>
      <widget class="QLabel" name="labelBoardSize">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Board size:</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="gameVariantInput">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>120</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string> Game variante</string>
       </property>
      </widget>
      <widget class="QComboBox" name="boardSizeInput">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>70</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">selection-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QLabel" name="ImgSettings">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>140</y>
        <width>261</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:qlineargradient(spread:repeat, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(255, 195, 2, 255), stop:0.816327 rgba(253, 155, 119, 255))</string>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>500</y>
        <width>236</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnGameOptions_play">
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGameOptions_mainMenu">
         <property name="text">
          <string>Main menu</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>241</width>
        <height>221</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../../../resources.qrc">:/resources/images/icon_settings.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="gamePage">
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>264</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4"/>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>540</y>
        <width>351</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Data current player</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>30</y>
         <width>331</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="labelMoveAvailable">
            <property name="font">
             <font>
              <pointsize>17</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:white;
</string>
            </property>
            <property name="text">
             <string>Moves available</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="movesAvailableNumber">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="smallDecimalPoint">
             <bool>false</bool>
            </property>
            <property name="digitCount">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="labelThrowAvailable">
            <property name="font">
             <font>
              <pointsize>17</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:white;</string>
            </property>
            <property name="text">
             <string>Throw available</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="throwAvailableNumber">
            <property name="digitCount">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>270</y>
        <width>351</width>
        <height>100</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelGameAction">
         <property name="styleSheet">
          <string notr="true">color:white;</string>
         </property>
         <property name="text">
          <string>GAME ACTION</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textActionGame">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>590</y>
        <width>314</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QPushButton" name="btnGamePage_passTurn">
         <property name="text">
          <string>Pass my turn</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGamePage_help">
         <property name="text">
          <string>Help </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGamePage_restart">
         <property name="text">
          <string>Restart</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="boardZone" native="true">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>621</width>
        <height>491</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelTurnTeam">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>110</y>
        <width>139</width>
        <height>26</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Turn of team</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="textTurnTeam">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>150</y>
        <width>291</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Gill Sans Nova Cond Ultra Bold</family>
        <pointsize>72</pointsize>
        <weight>10</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));

font: 87 72pt &quot;Gill Sans Nova Cond Ultra Bold&quot;;</string>
      </property>
      <property name="text">
       <string>TEAMX</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="winnerPage">
     <widget class="QLabel" name="ImgWinner">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>661</width>
        <height>291</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../../../resources.qrc">:/resources/images/winner_logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>380</y>
        <width>281</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelTeam">
         <property name="font">
          <font>
           <pointsize>35</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255))</string>
         </property>
         <property name="text">
          <string>TEAM</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="teamWinnerText">
         <property name="font">
          <font>
           <pointsize>35</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255))</string>
         </property>
         <property name="text">
          <string>TEAMX</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>580</y>
        <width>231</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="btnWinnerPage_mainMenu">
         <property name="text">
          <string>Main menu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnWinnerPage_exit">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="winReason">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>470</y>
        <width>481</width>
        <height>78</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>24</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));
font: 24pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>REASON</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="rulesPage">
     <widget class="QLabel" name="labelGamesRules">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:qlineargradient(spread:repeat, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(255, 195, 2, 255), stop:0.816327 rgba(253, 155, 119, 255))</string>
      </property>
      <property name="text">
       <string>Games rules</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRules_back">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>590</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Back </string>
      </property>
     </widget>
     <widget class="QLabel" name="ImgRules">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../../../resources.qrc">:/resources/images/rules_icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>140</y>
        <width>1041</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="labelFlowGame">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(254, 200, 9);</string>
         </property>
         <property name="text">
          <string>Flow of a game :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="TextFlowGame">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>The two opponents take turns. In turn, the player can perform up to 3 actions in any order: 2 orthogonal moves of a square and a pass. He is not obliged to perform all these actions. In order to move a player, the player must not have the ball and the adjacent square horizontally or vertically must be free (without a pawn of his team or of the opposing team). Since the player has two moves at his disposal, if he wants to move a pawn to an adjacent square diagonally, he must use two orthogonal moves, one vertically and one horizontally, with the passage square having to be free. It is also possible to move two different pieces to the same square. The pawn that has the ball can make a pass by throwing as far as it wants to another pawn of the same side on the same line, column or diagonal as itself, provided that no opposing pawn is placed between the two allied pawns. As in handball, the pawn with the ball may not move during the same turn (but the pawn receiving the ball may move before receiving the ball)</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelAntiGame">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(254, 200, 9);</string>
         </property>
         <property name="text">
          <string>Anti-Game :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="TextAntiGame">
         <property name="styleSheet">
          <string notr="true">color:white;</string>
         </property>
         <property name="text">
          <string>In order for each team to go to the opposing camp, an anti-play rule prohibits players, under certain conditions, from forming an impassable line for the opponent. If a player creates an impassable line and 3 opposing checkers are adjacent to this line, the player who made up the line loses immediately.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelVariante">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(254, 200, 9);</string>
         </property>
         <property name="text">
          <string>Variante :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textVariante">
         <property name="styleSheet">
          <string notr="true">color:white;</string>
         </property>
         <property name="text">
          <string>There is a variant that initially places two checkers in each team's opposing zone. This makes it faster to make attacks.
</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDiaballik_Game">
    <property name="title">
     <string>Diaballik Game</string>
    </property>
   </widget>
   <addaction name="menuDiaballik_Game"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
